apps/docs/src/guide/wallets/wallet-transferring.md

```diff
@@ -1,14 +1,59 @@
 # Wallet Transferring
 
-This guide demonstrates how to transfer assets between accounts and contracts, and how to validate your balance prior to a transfer.
+This guide provides instructions for transferring assets between wallets and contracts using the SDK. It includes methods for initiating transfers, handling transaction requests, and ensuring sufficient balance before transfers.
 
-## Transferring Between Wallets
+## Transferring Assets Between Accounts
 
-Transferring assets between wallets is really simple within the SDK.
+The `account.transfer` method initiates a transaction request that transfers an asset from one wallet to another. This method requires three parameters:
```

Comment:
```suggestion
The `transfer` method initiates a transaction request that transfers an asset from one wallet to another. This method requires three parameters:
```

I'd prefer to make this class agnostic due to the amount of inheritance we have and I don't think consumers will understand this complexity.